pyproject.toml

```diff
@@ -43,27 +43,6 @@ skip-string-normalization = true
 
 [tool.isort]
 include_trailing_comma = "True"
-known_third_party = [
-  "PIL",
-  "black",
-  "fairlearn",
-  "git",
-  "matplotlib",
-  "mypy",
-  "numpy",
-  "pandas",
-  "pylint",
-  "pytest",
-  "scipy",
-  "seaborn",
-  "setuptools",
-  "sklearn",
-  "tap",
-  "teext",
-  "torch",
-  "torchvision",
-  "tqdm",
-  "typing_extensions",
-]
+known_third_party = ["PIL", "black", "fairlearn", "git", "matplotlib", "mypy", "numpy", "pandas", "pylint", "pytest", "scipy", "seaborn", "setuptools", "sklearn", "tap", "teext", "torch", "torchvision", "tqdm", "typing_extensions"]
```

Comment:
my isort plugin only started doing this in the pyproject.toml. It didn't do it in isort.config.
Maybe there's a setting I have that's not right